config-service-factory/src/main/java/org/hypertrace/config/service/ConfigServiceFactory.java

```diff
@@ -91,6 +91,29 @@ localChannel, new LabelApplicationRuleConfig(config), configChangeEventGenerator
         .collect(Collectors.toUnmodifiableList());
   }
 
+  public List<GrpcPlatformService> buildServices(
```

Comment:
Dedupe this with above such that the above can construct a datastore and pass through to this one.